### PR TITLE
[TestMigration] Migration of test_snap_glusterd_down

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
[Test](https://github.com/gluster/glusto-tests/blob/master/tests/functional/snapshot/test_snap_glusterd_down.py)
    
Updates: #292
    
Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

**Comment by srijan-sivakumar** [comment]
```console
(redant-demo) [root@rhsdev-virt05 redant]# python3 core/redant_main.py -c ../rconfig.yml -t tests/functional/snapshot/test_snap_glusterd_down.py -ll D
    ____  __________  ___    _   ________
   / __ \/ ____/ __ \/   |  / | / /_  __/
  / /_/ / __/ / / / / /| | /  |/ / / /   
 / _, _/ /___/ /_/ / ___ |/ /|  / / /    
/_/ |_/_____/_____/_/  |_/_/ |_/ /_/     
                                         

✔ Param Handling Success.
✔ Test List built
✔ Log dir creation successful.
✔ Environment setup successful.
✔ Running test case : test_snap_glusterd_down-rep
✔ test_snap_glusterd_down-rep Succeeded
✔ Running test case : test_snap_glusterd_down-dist
✔ test_snap_glusterd_down-dist Succeeded
✔ Running test case : test_snap_glusterd_down-disp
✔ test_snap_glusterd_down-disp Succeeded
✔ Running test case : test_snap_glusterd_down-dist-rep
✔ test_snap_glusterd_down-dist-rep Succeeded
✔ Running test case : test_snap_glusterd_down-dist-disp
✔ test_snap_glusterd_down-dist-disp Succeeded
Results for snapshot
Test nature : disruptive
TestName : test_snap_glusterd_down
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:00:50       |      NA     |
|     dist    |     PASS    |        00:00:52       |      NA     |
|     disp    |     PASS    |        00:00:51       |      NA     |
|   dist-rep  |     PASS    |        00:01:11       |      NA     |
|  dist-disp  |     PASS    |        00:01:07       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   1   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 05 minutes 05 seconds
✔ Tearing down successful.
```

File: tests/functional/snapshot/test_snap_glusterd_down.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License along
+  with this program; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+Description:
+
+Test Cases in this module tests the
+snapshot activation and deactivation status
```
Comment (aujjwal-redhat) [inline tests/functional/snapshot/test_snap_glusterd_down.py:21]: NIT: indentation

**Review by aujjwal-redhat — COMMENTED** [review]
Other than the above everything looks fine.

File: tests/functional/snapshot/test_snap_glusterd_down.py (hunk 1 above)
Comment (srijan-sivakumar) [reply tests/functional/snapshot/test_snap_glusterd_down.py:21]: @aujjwal-redhat I think we can ignore this indentation...won't trouble out code logic 😛

Comment (aujjwal-redhat) [reply tests/functional/snapshot/test_snap_glusterd_down.py:21]: :sweat_smile: yeah no issues

**Review by aujjwal-redhat — APPROVED** [review]

**Pull request merged**